### DISCUSSION TITLE
Provide general PWS access

### DISCUSSION THREAD

**Discussion by DeckardCain2014** [cytech/Home-Assistant-wundergroundpws#161] [Ideas]
Hi,
coming from ioBroker, I was used to get data from any weather station, not my own (which I don't have). Would it be possible to use an access like in the ioBroker Addon (https://github.com/iobroker-community-adapters/ioBroker.weatherunderground) also in this here?

**Comment by nepozs** [comment]
ioBroker Addon uses a key obtained illegally (maybe semi-legally?).
Is there a lawyer here?
https://www.wunderground.com/company/legal

**Comment by cytech** [comment]
closed as outdated
